test/dlc_tests/container_tests/bin/testOpenCV.py

```diff
@@ -1,20 +1,28 @@
 import cv2
 import numpy as np
+import sys
 
-IMG_PATH='/test_img.jpg'
 
 
 def test_transformation(img):
+    print("Path of the image: ", img)
+    img = cv2.imread(img, cv2.IMREAD_UNCHANGED)
+    print('Original Dimensions: ', img.shape)
+
+    scale_percent = 60 # percent of original size
+    width = int(img.shape[1] * scale_percent / 100)
+    height = int(img.shape[0] * scale_percent / 100)
+    dim = (width, height)
+
     # Scaling
-    res = cv2.resize(img,None,fx=2, fy=2, interpolation = cv2.INTER_CUBIC)
+    res = cv2.resize(img,dim,fx=2, fy=2, interpolation = cv2.INTER_CUBIC)
     # Translation
-    rows,cols = img.shape
+    rows,cols,_= img.shape
     M = np.float32([[1,0,100],[0,1,50]])
     dst = cv2.warpAffine(img,M,(cols,rows))
     print('Successfully test OpenCV with image transformation')
 
 
 if __name__ == '__main__':
-    img = cv2.imread(IMG_PATH,0)
     print('CVtest: load the image')
```

Comment:
nit: This print could be moved to the location where the image is being actually loaded, i.e. inside the function `test_transformation` above line `img = cv2.imread(img, cv2.IMREAD_UNCHANGED)`

Reply:
ok, maybe it's better to create a separate pr for this

Reply:
@hsl89 can you keep track of this issue in a GitHub issue if you're planning to create a separate PR?